roles/nginxplus/files/conf/http/templates/pudl_proxy_pass.conf

```diff
@@ -1,307 +1,130 @@
-    }
-
-    rewrite ^/collections/pudl0102(.*)$ /pudl0102$1 redirect;
-    location /pudl0102 {
-        proxy_pass https://dpul.princeton.edu/music;
-    }
-
-    rewrite ^/collections/pudl0104(.*)$ /pudl0104$1 redirect;
-    location /pudl0104 {
-        proxy_pass https://dpul.princeton.edu/marquand-library;
-    }
-
-    rewrite ^/collections/pudl0106(.*)$ /pudl0106$1 redirect;
-    location /pudl0106 {
-        proxy_pass https://dpul.princeton.edu/pudl0106;
-    }
-
-    rewrite ^/collections/pudl0110(.*)$ /pudl0110$1 redirect;
-    location /pudl0110 {
-        proxy_pass https://dpul.princeton.edu/pudl0135;
-    }
-
-    rewrite ^/collections/pudl0112(.*)$ /pudl0112$1 redirect;
-    location /pudl0112 {
-        proxy_pass https://dpul.princeton.edu/islamicmss;
-    }
-
-    rewrite ^/collections/pudl0118(.*)$ /pudl0118$1 redirect;
-    location /pudl0118 {
-        proxy_pass https://dpul.princeton.edu/cotsen;
-    }
-
-    rewrite ^/collections/pudl0122(.*)$ /pudl0122$1 redirect;
-    location /pudl0122 {
-        proxy_pass https://dpul.princeton.edu/pudl0135;
-    }
-
-    rewrite ^/collections/pudl0123(.*)$ /pudl0123$1 redirect;
-    location /pudl0123 {
-        proxy_pass https://dpul.princeton.edu/msstreasures;
-    }
-
-    rewrite ^/collections/pudl0125(.*)$ /pudl0125$1 redirect;
-    location /pudl0125 {
-        proxy_pass https://dpul.princeton.edu/slavic;
-    }
-
-    rewrite ^/collections/pudl0127(.*)$ /pudl0127$1 redirect;
-    location /pudl0127 {
-        proxy_pass https://dpul.princeton.edu/cotsen;
-    }
-
-    rewrite ^/collections/pudl0129(.*)$ /pudl0129$1 redirect;
-    location /pudl0129 {
-        proxy_pass https://dpul.princeton.edu/cotsen;
-    }
-
-    rewrite ^/collections/pudl0130(.*)$ /pudl0130$1 redirect;
-    location /pudl0130 {
-        proxy_pass https://dpul.princeton.edu/ga_treasures;
-    }
-
-    rewrite ^/collections/pudl0137(.*)$ /pudl0137$1 redirect;
-    location /pudl0137 {
-        proxy_pass https://dpul.princeton.edu/pudl0017;
-    }
-
-    rewrite ^/collections/pudl0139(.*)$ /pudl0139$1 redirect;
-    location /pudl0139 {
-        proxy_pass https://dpul.princeton.edu/cotsen;
-    }
+    rewrite ^/search.php(.*)$ https://dpul.princeton.edu/search.php$1 redirect;
+
+    rewrite ^/collections.php(.*)$ https://dpul.princeton.edu$1 redirect;
+
+    rewrite ^/about.php(.*)$ https://dpul.princeton.edu$1 redirect;
+
+    rewrite ^/collections/eal(.*)$ https://dpul.princeton.edu/eastasian$1 redirect;
+
+    rewrite ^/collections/garden(.*)$ https://dpul.princeton.edu/marquand-library$1 redirect;
+
+    rewrite ^/collections/papyri(.*)$ https://dpul.princeton.edu/papyri$1 redirect;
+
+    rewrite ^/collections/paris(.*)$ https://dpul.princeton.edu/marquand-library$1 redirect;
+
+    rewrite ^/collections/scheide(.*)$ https://dpul.princeton.edu/scheide$1 redirect;
+
+    rewrite ^/collections/versailles2$ https://dpul.princeton.edu/ga_treasures$1 redirect;
```

Comment:
Thanks for splitting out the two versailles patterns — now I don't see a `(.*)` pattern in them, so is the `$1` in the destination still needed?

Reply:
probably not... incidentally the proxy_pass was not necessary (sorry about that). I've made the name switch so you can test these and comment on broken items here.